astropy/modeling/powerlaws.py

```diff
@@ -454,6 +454,120 @@ def _parameter_units_for_data_units(self, inputs_unit, outputs_unit):
                             ('amplitude', outputs_unit['y'])])
 
 
+class Schechter1D(Fittable1DModel):
+    """
```

Comment:
Please use raw stings here. (``"""`` --> ``r"""``)

python 3.6 switched to stricter string parsing, so as a rule of thumb we need to use raw strings whenever there is any math in the docstring.